plonky2/src/fri/batch_prover.rs

```diff
@@ -112,16 +112,17 @@ pub(crate) fn batch_fri_committed_trees<
         if polynomial_index != values.len() && final_values.len() == values[polynomial_index].len()
         {
             final_values = PolynomialValues::new(
-                final_values
-                    .values
-                    .iter()
-                    .zip(&values[polynomial_index].values)
-                    .map(|(&f, &v)| f * beta + v)
+                zip_eq(final_values.values.iter(), &values[polynomial_index].values)
+                    .map(|(&f, &v)| f + v)
+                    .collect::<Vec<_>>(),
+            );
+            final_coeffs = PolynomialCoeffs::new(
+                zip_eq(final_coeffs.coeffs.iter(), &coeffs[polynomial_index].coeffs)
+                    .map(|(&f, &v)| f + v)
```

Comment:
In fact we can use a challenge of our choice here to do things like `f + alpha * v`. This is just proof of concept that `f`  and `v` are in correct domains.

Reply:
Let's leave the multiplication `f * beta + v` in for now.

(Both for values and coefficients.)